test/WebJobs.Script.Tests.Integration/WebHostEndToEnd/SamplesEndToEndTests_CSharp.cs

```diff
@@ -98,6 +98,37 @@ public async Task HostPing_Succeeds(string method)
             Assert.Equal("no-store, no-cache", cacheHeader);
         }
 
+        [Fact]
+        public async Task SyncTriggers_InternalAuth_Succeeds()
+        {
+            using (new TestScopedSettings(_settingsManager, EnvironmentSettingNames.AzureWebsiteInstanceId, "testinstance"))
```

Comment:
internal auth only allowed in AppService, so we have to set the instance ID here to simulate that